src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Identity
+{
+    public class IdentityLogger
+    {
+        public IdentityLogger(ILogger logger)
+        {
+            Logger = logger;
+        }
+
+        protected internal virtual ILogger Logger { get; set; }
+
+        public virtual TResult Log<TResult>(TResult result, Func<TResult, LogLevel> getLevel,
```

Comment:
I  was a slightly ambiguous on this since if ever they write a method that returns anything else than IdentityResult, SignInResult or bool, they could use it directly. Sure making protected lets them subclass and use it which is fine too